week-2/0-practice/1-finish-classwork.md

```diff
@@ -2,4 +2,4 @@
 
 If you haven't already, complete the in-class exercises on your pokedex app.
 
-You can find the classwork [here](https://syllabus.codeyourfuture.io/react/week-2/lesson.html).
+You can find the classwork [here](https://syllabus.codeyourfuture.io/react/week-2/lesson).
```

Comment:
```suggestion
You can find the classwork [on the syllabus](https://syllabus.codeyourfuture.io/react/week-2/lesson).
```

[links must make sense out of context ](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)